jsuite/count.py

```diff
@@ -8,4 +8,8 @@ def of_type_in_body(etree, type):
 
 def of_type(etree, type):
   xpath_query = './/%s' %(type)
-  return len(etree.findall(xpath_query))
\ No newline at end of file
+  return len(etree.findall(xpath_query))
+
+def of_type_contrib(etree):
```

Comment:
Better named as of_authors, because JATS allows other contrib elements to exist.

Reply:
Also, please add test cases. There are enough examples to create one.

Reply:
But this one gets the count of contrib and not the author